### PR TITLE
Feat/migrate to gh actions

### PR DESCRIPTION
**Pull request by tunderwood**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      - run: npm run test:unit:local
+      - name: Upload coverage report to CodeCov
+        uses: codecov/codecov-action@v1
+        with:
+          file: coverage/clover.xml # optional
+          fail_ci_if_error: true # optional (default = false)
+          verbose: true
+      - name: Publish to NPM
+          uses: pascalgn/npm-publish-action@1.3.4
+          with: # All of theses inputs are optional
```
Comment (melanahammel) [inline .github/workflows/release.yml:38]: Does this comment mean the command works without the inputs?

File: README.md [hunk 2]
```diff
@@ -6,7 +6,8 @@ To learn more about Cerberus, please visit the [Cerberus website](http://enginee
 
 This library is compatible with node 8.x +, for previous versions of Node use the [1.x versions of the client](https://github.com/Nike-Inc/cerberus-node-client/tree/1.X_kms_client)
```
Comment (melanahammel) [inline README.md:7]: We aren't building for node 8 it seems, so is this comment out of date?

File: README.md [hunk 3]
```diff
@@ -21,7 +22,7 @@ See the [CerberusClient](http://engineering.nike.com/cerberus-node-client/Cerber
 ## Authentication
 
 The cerberus client uses the [AWS SDK Credentials provider chain][node aws credentials] to load AWS IAM credentials and authenticates with Cerberus via the [sts auth endpoint](https://github.com/Nike-Inc/cerberus-management-service/blob/master/API.md#app-login-sts-v2-v2authsts-identity)
-This client will work in any environment that has access to AWS Credentials.
+This client will work in any vironment that has access to AWS Credentials.
```
Comment (melanahammel) [inline README.md:25]: typo: `vironment` should be `environment`

File: README.md [hunk 4]
```diff
@@ -42,7 +43,7 @@ gimme-aws-creds
 
 Next, in the project directory run: 
 ```
-npm run style && npm run test:unit:local && npm run test:integration
+npm run stylene && npm run test:unit:local && npm run test:integration
```
Comment (melanahammel) [inline README.md:46]: Also a typo? What is `stylene`?

**Review by melanahammel — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (tunderwood) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (tunderwood) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (tunderwood) [reply README.md]: fixed

File: .github/workflows/release.yml (hunk 1 above)
Comment (tunderwood) [reply .github/workflows/release.yml:38]: Yes, it does. the comment was from the example code I copy/pasted showing how to use the plugin. There are defaults for all these inputs and we are overriding them for our own usecase.

**Review by rdong333 — APPROVED** [review]

**Pull request merged**